awacs/__init__.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, name, type=None, dictname=None, props={}, **kwargs):
         self.name = name
         self.props = props
         # Cache the keys for validity checks
-        self.propnames = props.keys()
+        self.propnames = list(props.keys())
```

Comment:
Is `list()` necessary here?  The `in` check in line 50 doesn't need it.
https://docs.python.org/3/library/stdtypes.html#dict-views

Reply:
`AWSObject` and `AWSHelperFn` can also stop inheriting from `object` explicitly.

Reply:
fixed